src/main/java/com/zoomcare/candidatechallenge/repositories/EmployeeRepository.java

```diff
@@ -0,0 +1,96 @@
+package com.zoomcare.candidatechallenge.repositories;
+
+import com.zoomcare.candidatechallenge.Models.*;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.PropertyPermission;
+
+/*
+The data structure in the database lends itself nicely to JSON objects rather than models
+but the JDBC and SQL really don't.  I probably would rather do JSON but my initial
+attempts made it appear a bit rougher than using models; if the app were going to contain
+many many models, I would push very hard to use JSON instead.
+ */
+@Repository
+public class EmployeeRepository {
```

Comment:
This should have an interface and implement the interface for decoupling from the data source.